main/main.c

```diff
@@ -554,6 +554,14 @@ void interactiveLoop (cookedArgs *args CTAGS_ATTR_UNUSED, void *user CTAGS_ATTR_
 			openTagFile ();
 			if (size == -1)
 			{					/* read from disk */
+#ifdef HAVE_SECCOMP
+				struct interactiveModeArgs *iargs = user;
+				if (iargs->secure) {
+					error (FATAL,
+						   "invalid request in secure submode: reading file contents from a file is limited");
+					goto next;
```

Comment:
actually, this is not needed at all. seccomp will kill the process anyway with a special signal, and this would be useful to test that it works.